Web Site Pages/SRVvsIRV.txt

```diff
@@ -15,25 +17,47 @@ Traditional Instant Runoff Voting (IRV) is the Ranked Choice O.G. - it's been us
 
 <p><img src="irv_srv_ballots.png" alt="irv_srv_ballots.png" /></p>
 
-IRV is one of many forms of Ranked Choice Voting that use a preference order ballot: first choice, second choice, third choice and so on. Because full rank ordering makes for a complex ballot (basically an N x N choice matrix for each office), IRV proponents typically advocate a limit of three or four choices on the ballot. SRV is a form of Ranked Choice that uses a score ballot: the voter can assign a level of support for each candidate, from zero to the maximum score.
+<h4>IRV uses rank-order ballot</h4>
+<p>
+IRV is one of many forms of Ranked Choice Voting that use a preference order ballot: first choice, second choice, third choice and so on. Because full rank ordering makes for a complex ballot (basically an N x N choice matrix for each office), IRV proponents typically advocate a limit of three or four choices on the ballot.
+</p>
+
+<h4>SRV uses a score ballot</h4>
 
+<p>
+SRV is a form of Ranked Choice that uses a score ballot: the voter can assign a level of support for each candidate, from zero to the maximum score.
+</p>
+
+<h4>Who gets to the final runoff?</h4>
 <p>Both IRV and SRV have a final instant runoff round to determine the winner. What differentiates the two system is how they determine which candidates are present in the final runoff.
+</p>
+
+<h4>IRV uses multi-round elimination</h4>
 
-<p>In IRV, the final contestants are determined by successively eliminating the candidate with the fewest first-choice votes amongst the voters. With each elimination, the votes of those who chose the eliminated candidate in top position transfer to the next choice on the ballot, until the final instant runoff that gives one candidate a majority of first choice votes amongst the ballots that remain.
+<p>IRV determines the final contestants through a series of elimination rounds. In the first round, IRV counts only the top choice from each ballot. If no candidate has a majority, then the candidate with the fewest top votes gets eliminated. The ballots for that candidate are moved to their 2nd choice (if any is marked). In each successive round, the candidate with the fewest ballots in their column gets eliminated. Ballots from that column then move to their next choice among the candidates still remaining (if any). Once some candidate has a majority of the still-active ballots, they win the election.</p>
```

Comment:
This line has wording updates that I like and think are valuable to be considered for inclusion still (given that some things like headers have been copied over already)